### ISSUE TITLE
twisted/web/_newclient.py uses invalid type in docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5602]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5602|
|Type|defect|
|Created|2012-04-02 21:10:28Z|

`Request.persistent` is documented with _@type persistent: Boolean_

_Boolean_ is not a Python type.  This should be `C{bool}`.

Attachments:

* [5602.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/aef/aefbeaff6384420840c59a0c1c6a31bcba8e133f/9623ba4e67edbe5b67ccf80a022f19b63865021b.patch) (873 bytes) - added by chhabraamit on 2012-04-06 10:23:11Z - 
* [5602.misc](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/aef/aefbeaff6384420840c59a0c1c6a31bcba8e133f/a591fc060d2a15bf77afc69ce46fcd310272e5e5.misc) (123 bytes) - added by chhabraamit on 2012-04-06 10:23:32Z - 
* [5602.2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/aef/aefbeaff6384420840c59a0c1c6a31bcba8e133f/1698fdf5005c2539ea63a20ca5876522a1d62061.patch) (888 bytes) - added by chhabraamit on 2012-04-06 12:00:09Z - 
* [5602.2.misc](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/aef/aefbeaff6384420840c59a0c1c6a31bcba8e133f/1af1b40cba4107f715170f80bf7a0e86442f38a5.misc) (127 bytes) - added by chhabraamit on 2012-04-06 12:00:46Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5602 5602
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1333401028000000 1333401028000000
changetime__1333819562000000 1333819562000000
version__None None
owner__thijs thijs
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for the patch. I'll appply 5602.2.patch but use an empty .misc file, as explained on ReviewProcess#Newsfiles.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34126]) Apply 5602.2.patch: Correct epytext for `persistent` in the docstring of t.web._newclient.Request.

Author: chhabraamit
Reviewer: thijs
Fixes: [#5602](https://github.com/twisted/twisted/issues/5602)
